game/world/managers/objects/units/UnitManager.py

```diff
@@ -102,6 +102,7 @@ def __init__(self,
         self.combat_target = combat_target
         self.channel_spell = channel_spell
         self.channel_object = channel_object
+        self.quest_target = None
```

Comment:
Maybe this can be moved to CreatureManager instead of UnitManager? And I guess add a `# TODO: Move this to a new EscortAI that needs to be created`, in the future it will be cleaner to handle this kind of escort behaviors with a different class.